packages/strapi-plugin-content-manager/admin/src/components/InputUID/Options/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import pluginId from '../../../pluginId';
+import OptionsWrapper from './wrapper';
+import Option from './Option';
+import OptionsTitle from './OptionsTitle';
+import RightOptionLabel from './RightOptionLabel';
+
+const Options = ({ options, title }) => {
+  const [displayRightOptionLabel, setDisplayRightOptionLabel] = useState(false);
+  const handleOptionMouseEnter = () => {
+    setDisplayRightOptionLabel(true);
+  };
+  const handleOptionMouseLeave = () => {
+    setDisplayRightOptionLabel(false);
+  };
+
+  return (
+    <OptionsWrapper>
+      {title && <OptionsTitle>{title}</OptionsTitle>}
+      {options.map(option => (
+        <Option
+          onMouseEnter={handleOptionMouseEnter}
+          onMouseLeave={handleOptionMouseLeave}
+          key={option.id}
+          onClick={option.onClick}
+        >
+          <div>{option.label}</div>
+          {displayRightOptionLabel && (
+            <FormattedMessage id={`${pluginId}.components.uid.apply`}>
```

Comment:
You can use the `getTrad` util here unless you want to avoid using functions in the render method.